test/spec/asyncapi.yml

```diff
@@ -145,6 +145,7 @@ components:
           x-pi: false
         sentAt:
           $ref: "#/components/schemas/sentAt"
+      additionalProperties: false
```

Comment:
There are currently no allOf usages in the template. I wasn't sure on the best example to add here.